extension/lib/settings_view.js

```diff
@@ -102,8 +102,9 @@ class SettingsView {
         this.keywordSetting = keywordSetting;
         this.highlighterSetting = highlighterSetting;
         this.shadowSetting = shadowSetting;
-        trackerStyle.setKeywordStyle("keyWord"+keywordSetting);
+        // TODO: It shouldn't have to call both of these; in future only call whichever is needed
```

Comment:
hm i dont understand this change, looks like its just swapping the order of the two lines?
or is the main point the todo comment?

Reply:
Eh yeah the main point is the todo, the swap of order is trivial. I want to in general order the sentence stuff before keyword, but right now there's not a universal precedent so it's not important [see: above, even within this function]